extensions.json

```
@@ -1,5 +1,10 @@
 {
   "extensions": [
+    {
+      "id": "small.php-ci",
+      "repository": "https://github.com/smallp/vscode-ci",
+      "version": "0.4.1"
+    },
```

When I run `node add-extension https://github.com/smallp/vscode-ci --checkout`, I get this instead:

```suggestion
    {
      "id": "small.php-ci",
      "download": "https://github.com/smallp/vscode-ci/releases/download/0.4.2/php-ci-0.4.2.vsix",
      "version": "0.4.2"
    },
```